packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -138,6 +141,12 @@ export const RemoteServerFactory = {
         // This is the ORIGIN under which we are serving the page.
         // because it's an origin, it does not end with a slash.
         options.appOrigin = process.env.APP_ORIGIN = `${options.protocol}://${options.appHostname}`
+
+        if ('SSR_ALLOW_COOKIES' in process.env) {
```

Comment:
superr nit: based on this [discussion](https://salesforce-internal.slack.com/archives/C055R5628SX/p1687896986746829?thread_ts=1687896910.315899&cid=C055R5628SX) we could strip the `ssr_` prefix everywhere and just use `allow_cookies` throughout the codebase

Reply:
As per our Slack discussion, we'll want to use a reserved prefix here to avoid clashes with user provided environment variables.

A good name would `MRT_ALLOW_COOKIES` as the MRT prefix is reserved and there is emerging precedent for using it in the code.